src/common/transformations/src/transformations/op_conversions/convert_avgpool_downgrade.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_avgpool_downgrade.hpp"
+
+#include "itt.hpp"
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/avg_pool.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/pad.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/utils/utils.hpp"
+
+ov::pass::ConvertAvgPool14ToAvgPool1::ConvertAvgPool14ToAvgPool1() {
+    MATCHER_SCOPE(ConvertAvgPool14ToAvgPool1);
+
+    const auto avg_pool_v14_pattern = pattern::wrap_type<ov::op::v14::AvgPool>();
+
+    const matcher_pass_callback callback = [](pattern::Matcher& m) {
+        const auto avg_pool_v14 = std::dynamic_pointer_cast<ov::op::v14::AvgPool>(m.get_match_root());
+        const auto rounding_type_v14 = avg_pool_v14->get_rounding_type();
+        const auto rounding_type_v1 =
+            rounding_type_v14 == ov::op::RoundingType::CEIL_TORCH ? ov::op::RoundingType::CEIL : rounding_type_v14;
+
+        const auto exclude_pad = avg_pool_v14->get_exclude_pad();
+        const auto input = avg_pool_v14->input_value(0);
+        NodeRegistry node_registry;
+        ov::Shape pads_begin;
+        ov::Shape pads_end;
+
+        using ov::op::v0::Constant;
+        using ov::op::v0::Concat;
+        using ov::op::v1::Pad;
+        using ov::op::v1::Subtract;
+        using ov::op::v1::ConvertLike;
+        using ov::op::v3::Broadcast;
+        using ov::op::v3::ShapeOf;
+
+        if (!exclude_pad) {
```

Comment:
Should this case be enabled for `ov::op::RoundingType::CEIL_TORCH`?
```suggestion
        if (!exclude_pad && rounding_type_v14 == ov::op::RoundingType::CEIL_TORCH) {
```

Reply:
I agree, condition added.